scripts/mbedtls_framework/min_requirements.py

```diff
@@ -111,19 +109,16 @@ def main() -> None:
                              " (short for --pip-install-option --user)")
     parser.add_argument('files', nargs='*', metavar='FILE',
                         help="Requirement files"
-                             " (default: {} in the script's directory)" \
-                             .format(DEFAULT_REQUIREMENTS_FILE))
+                             " (default: {})" \
+                             .format(default_requirement_file))
     options = parser.parse_args()
     if not options.files:
         options.files = [os.path.join(os.path.dirname(__file__),
-                                      DEFAULT_REQUIREMENTS_FILE)]
+                                      default_requirement_file)]
```

Comment:
Currently we pass an absolute path for `default_requirement_file` and if we were to pass a relative path it would be rather relative to the directory of the caller I'd say thus I am a bit confused by this `path.join`.

Reply:
Ops, sorry, this is now somehow replicating the work the of the main scripts indeed. Thanks for noticing